doc/man1/openssl.pod

```diff
@@ -392,7 +392,7 @@ SM3 Digest
 
 =back
 
-=head2 Encoding and Cipher Commands
+=head2 Encryption, Decryptions, and Encoding Commands
```

Comment:
Decryption

Reply:
"decryptions" sounds so steam-punk doesn't it ?  fixed, thanks.